pr_agent/servers/github_action_runner.py

```diff
@@ -16,6 +16,7 @@ async def run_action():
     GITHUB_EVENT_PATH = os.environ.get('GITHUB_EVENT_PATH')
     OPENAI_KEY = os.environ.get('OPENAI_KEY') or os.environ.get('OPENAI.KEY')
     OPENAI_ORG = os.environ.get('OPENAI_ORG') or os.environ.get('OPENAI.ORG')
+    BITO_CLI_KEY = os.environ.get('BITO_KEY') or os.environ.get('BITO.KEY')
     GITHUB_TOKEN = os.environ.get('GITHUB_TOKEN')
     get_settings().set("CONFIG.PUBLISH_OUTPUT_PROGRESS", False)
 
```

Comment:
**Suggestion**: The introduction of 'BITO_CLI_KEY' as an environment variable is a significant addition. Ensure that its usage is documented, and provide examples or further information in the project's README or relevant documentation. <br> **Code Suggestion**: 
 ```
 # Documentation should be updated to reflect the introduction of BITO_CLI_KEY
 ```